app/components/special-coverage/special-coverage-edit/special-coverage-edit.html

```diff
@@ -107,9 +107,8 @@
         <label for="specialCoverageCampaign">Sponsor Campaign</label>
         <autocomplete-basic
             class="form-control"
-            ng-if="!model.campaign || model.campaign.$resolved"
```

Comment:
If I leave that in, an existing campaign will not be rendered:

https://s3.amazonaws.com/onionscreenshots/captured//0KogB.png